lifecycle.md

```diff
@@ -0,0 +1,61 @@
+# Prereq container image lifecycle
+
+The container images in this repository are provided for building .NET source and testing the resultant binaries. The images are built with OS base images and must be updated per the OS lifecycle. This document describes the required patterns.
+
+The images are a mix of Microsoft- and community-supported images.
+
+## Build images
+
+The build images are primarily built with Azure Linux, supporting both `glibc` and `musl` build flavors. They are built for x64 only, per [Linux build methodology](https://github.com/dotnet/runtime/blob/main/docs/project/linux-build-methodology.md). Microsoft designates the set of images it used for its official build per [The Official Runtime Docker Images](https://github.com/dotnet/runtime/blob/main/docs/workflow/using-docker.md#the-official-runtime-docker-images).
+
+Build images are uniquely created for each new .NET version with the latest Azure Linux available at that time. The image tags include both the given .NET and Azure Linux versions. The tag may also contain a target OS and architecture.
```

Comment:
This also does not hold for community images today. Maybe this whole chapter should be about `Microsoft official build images" only.

Reply:
Happy to. If we do that, we should add some text on the community images, too.

I got a bit confused because FreeBSD seems to be fully integrated: https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/1272/files.